### ISSUE TITLE
Accessibility for screen reader users

### ISSUE THREAD

**Issue by mjanusauskas** [chinchang/web-maker#61]
The contents of the input boxes of the editor can not be reviewed or navigated by screen reader users. They report themselves as blank.
Web-Maker looks like it could be a really great and useful tool if only it could be made more accessible and inclusive so everyone that wished to use it could do so.

**Comment by chinchang** [comment]
@mjanusauskas By input boxes, you mean the actual code editing area?
Also, what would be the expected behaviour here...should screen reader read the complete code?

**Comment by mjanusauskas** [comment]
Yes I was referring to the code editing area. If you press CTRL+a to select
all contents then the entire content is spoken by the screen reader which
is not useful. The user needs to be able to actually review and edit the
content. Typical screen reader behavior would be when navigating through
the text (moving the input cursor) the character / word / line would be
spoken as the cursor moved to it.
The free and open source NVDA screen reader for Windows and the built-in
VoiceOver screen reader on Mac OS allow for some easy testing tools.
If you launch one of those when composing an email in Gmail for example you
can observe the expected  behavior.
(Remember most screen reader users rely on keyboard navigation exclusively
as they are not able to use a mouse.)
